catkin_ws/src/kpr_interface/cfg/settings.json

```diff
@@ -0,0 +1 @@
+{"maxCurvature": "100", "maxWeight": "4400", "minWeight": "300", "armSpeed": "100"}
```

Comment:
Why do you save these as strings? JSON does support numbers (in this case floats I suppose)